Source_Code/Client/ClueLessClient.java

```diff
@@ -342,11 +344,20 @@ else if(statUp instanceof Notification)  // Generic server print essentially
         }
         else if(statUp instanceof GameStart)  // Starting/ending the game
         {
-            if(((GameStart) statUp).GameStarting)
+            if(((GameStart) statUp).GameStarting && !activeGame )
             {
                 System.out.println("[Server] Game starting!\n");
+                
+                // Only instantiate the board after the game starts
+                this.board = new Board();
                 activeGame = true;
             }
+            
+            else if( activeGame )
+            {
+               System.out.println("[Server] Game has already been started!\n");
```

Comment:
Don't know that we want/need to show this to the user, although I don't think we ever plan to send these out so it probably doesn't matter